ionic2/app/components/evolutions/evolutions.component.scss

```diff
@@ -10,8 +10,10 @@
 }
 
 .evolutions > div > button {
-  width: 100px;
-  height: 100px;
+  max-width: 100px;
+  max-height: 100px;
+  width: 23vw;
+  height: 23vw;
```

Comment:
IMO percentages are better than viewport width & height. That way the sizes of child elements are relative to the component's size.